src/skolepenger/Forside.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react';
+import { Panel } from 'nav-frontend-paneler';
+import { Sidetittel } from 'nav-frontend-typografi';
+import { usePersonContext } from '../context/PersonContext';
+import { useSpråkContext } from '../context/SpråkContext';
+import { hentBeskjedMedNavn } from '../utils/språk';
+import { injectIntl } from 'react-intl';
+import { client } from '../utils/sanity';
+import VeilederSnakkeboble from '../arbeidssøkerskjema/VeilederSnakkeboble';
+import { useSkolepengerSøknad } from './SkolepengerContext';
+import Environment from '../Environment';
+import Forsideinformasjon from './Forsideinformasjon';
+import FortsettSøknad from '../søknad/forside/FortsettSøknad';
+
+const Forside: React.FC<any> = ({ intl }) => {
+  const { person } = usePersonContext();
+  const [locale] = useSpråkContext();
+  const {
+    mellomlagretSkolepenger,
+    brukMellomlagretSkolepenger,
+    nullstillMellomlagretSkolepenger,
+  } = useSkolepengerSøknad();
+  const [forside, settForside] = useState<any>({});
+  // eslint-disable-next-line
+  const [error, settError] = useState<boolean>(false);
+  // eslint-disable-next-line
+  const [fetching, settFetching] = useState<boolean>(false);
+
+  useEffect(() => {
+    const fetchData = () => {
+      client
+        .fetch('*[_type == $type][0]', { type: 'forside_barnetilsyn' }) // TODO: Må opprettes i sanity
+        .then((res: any) => {
+          settForside(res);
+        })
+        .catch((err: any) => {
+          settError(true);
+        });
+      settFetching(false);
+    };
+    fetchData();
+  }, []);
+
+  const kanBrukeMellomlagretSøknad =
+    mellomlagretSkolepenger !== undefined &&
+    mellomlagretSkolepenger.søknad.person.hash === person.hash &&
+    mellomlagretSkolepenger.modellVersjon ===
+      Environment().modellVersjon.skolepenger;
+
+  const disclaimer = forside['disclaimer_' + locale];
+  const seksjon = forside['seksjon_' + locale];
+
+  return (
+    <div className={'forside'}>
+      <main className={'forside__innhold'}>
+        <Panel className={'forside__panel'}>
+          <div className="veileder">
+            <VeilederSnakkeboble
+              tekst={hentBeskjedMedNavn(
+                person.søker.forkortetNavn,
+                intl.formatMessage({ id: 'skjema.hei' })
+              )}
+            />
+          </div>
+          <Sidetittel>
+            Søknad om stønad til skolepenger for enslig mor eller far i arbeid
```

Comment:
Her kunne vi brukt `<LocaleTekst tekstid={''}` eller `intl.formatMessage({ id: ''}) for teksten :)